src/parse_file.py

```diff
@@ -164,7 +169,7 @@ def parse_ascii_directive(line: str, z: bool) -> bytes:
 
 
 def parse_file(f: typing.TextIO, options: Options) -> MIPSFile:
-    filename = f.name
+    filename = Path(f.name).name
```

Comment:
This was changed to remove the full filepath from the error message -- now it only prints the basename instead of the full path. This would make the test reproducible on machines with different paths. I could punt this to a later PR if you think keeping the full path is worth it.

Example:
```diff
-Found jr instruction at /home/zbanks/mm/asm/code/code_0x80161180.asm line 522, but the corresponding jump table is not provided.
+Found jr instruction at code_0x80161180.asm line 522, but the corresponding jump table is not provided.
```

Reply:
ah, makes sense